ludwig/schema/decoders/base.py

```diff
@@ -76,7 +76,7 @@ class PassthroughDecoderConfig(BaseDecoderConfig):
 class RegressorConfig(BaseDecoderConfig):
     """RegressorConfig is a dataclass that configures the parameters used for a regressor decoder."""
 
-    type: str = schema_utils.StringOptions(
+    type: str = schema_utils.ProtectedString(
         ["regressor"],
         default="regressor",
         allow_none=False,
```

Comment:
@ksbrar I think this should likely be updated to

```
type: str = schema_utils.ProtectedString("regressor", description="...")
```

Reply:
woops :)